policyengine_us/variables/gov/states/az/tax/income/deductions/itemized/az_itemized_deduction.py

```diff
@@ -0,0 +1,94 @@
+from policyengine_us.model_api import *
+
+
+class az_itemized_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Itemized Deduction Adjustments"
+    unit = USD
+    documentation = "Arizona Form 140 Schedule A"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+    reference = (
+        "https://azdor.gov/forms/individual/itemized-deduction-adjustments-form"
+        "https://azdor.gov/forms/individual/form-140-resident-personal-income-tax-form-calculating"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        # Adjustment to Medical and Dental Expresses
+        medical_expense = add(tax_unit, period, ["medical_expense"])
+        medical = parameters(period).gov.irs.deductions.itemized.medical
+        medical_expense_irs = medical.floor * tax_unit("positive_agi", period)
+        medical_expense_larger_then_irs_allowed = where(
+            medical_expense >= medical_expense_irs,
+            medical_expense - medical_expense_irs,
+            0,
+        )
+        medical_expense_less_then_irs_allowed = where(
+            medical_expense_irs > medical_expense,
+            medical_expense_irs - medical_expense,
+            0,
+        )
+
+        # Adjustment to Interest Deduction
+        # mortgage_interest = the amount of mortgage interest you paid for 2022 that is equal to
+        # the amount of federal credit, if you received a federal credit for interest paid on
+        # mortgage credit certificates (from federal Form 8396).
+        # adjusrment_intertest_deduction = 0
+
+        # Adjustments to Charitable Contributions
+        # Amount of charitable contributions for which you are claiming a credit under Arizona law
+        # I assume it is the same as irs after discussing with Pavel
+        charitable_deduction = tax_unit("charitable_deduction", period)
```

Comment:
we can actually model the credit - we can either make a placeholdr variable or model the credit under these sections:
https://casetext.com/statute/arizona-revised-statutes/title-43-taxation-of-income/chapter-10-individuals/article-5-credits/section-43-1088-effective-until-ninety-one-days-after-adjournment-credit-for-contribution-to-qualifying-charitable-organizations-definitions